doc/index.md

```diff
@@ -101,6 +101,21 @@ group to talk to LXD; you can create your own group if you want):
 sudo -E LD_LIBRARY_PATH=$LD_LIBRARY_PATH $GOPATH/bin/lxd --group sudo
 ```
 
+## Security
+LXD, similar to other container managers provides a UNIX socket for local communication.
+
+**WARNING**: Anyone with access to that socket can fully control LXD, which includes
+the ability to attach host devices and filesystems, this should
+therefore only be given to users who would be trusted with root access
+to the host.
+
+When listening on the network, the same API is available on a TLS socket
+(HTTPs), specific access on the remote API can be restricted through
```

Comment:
@stgraber HTTPS? rather than lowercase s?

Reply:
fixed and put a follow-up commit that brings the rest of the docs in line